docs/ros/ros/tutorials/simulator/overview.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Overview
+sidebar_label: Overview
+sidebar_position: 1
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+The Clearpath simulator uses [Gazebo Fortress](https://gazebosim.org/home) to simulate robots in a physics-based environment. The simulation
+supports the Clearpath config, as well as simulating multiple robots simultaneously. Gazebo data is bridged to ROS 2, and follows the [Clearpath API](../api/overview).
```

Comment:
Can we make "Clearpath Config" a defined term, so capitalized on both words, possibly with a link to your definitions page?